Cargo.toml

```diff
@@ -6,8 +6,8 @@ members = [
     "pg-extend", 
     "pg-extern-attr", 
     "examples/adding", 
-    "examples/fdw",
-    "examples/fdw-rw",
+#    "examples/fdw",
```

Comment:
At the very top of the `lib.rs` and/or `bin.rs` how about just adding `#![cfg(not(feature = "postgres-12"))]` Alterntatively, we could list out all the features that each version supports in the Cargo.toml, and use a new flag there for `fdw`...